### ISSUE TITLE
get_output

### ISSUE THREAD

**Issue by DELTA37** [DELTA37/TimeSeries#19]
Create a get_output method to replace data['label'] in train.py

**Comment by DELTA37** [comment]
https://github.com/pytorch/pytorch/pull/2037#pullrequestreview-49643745  
we faced with an open issue on torch
so i have commented the future code in 6e3ed46
